app/src/main/res/values-ru-rRU/strings.xml

```diff
@@ -0,0 +1,422 @@
+    <string name="dismiss">Отклонить</string>
+
+    <string-array name="nav_list_titles">
+        <item>Обзор</item>
+        <item>История</item>
+        <item>Отправить</item>
+        <item>Получить</item>
+        <item>Аккаунты</item>
+        <item>Безопасность</item>
+        <item>Настройки</item>
+        <item>Справка</item>
+    </string-array>
+    <!--/General-->
+
+    <!--Peers-->
+    <string name="view_peers">Просмотреть узлы</string>
+    <string name="services">Сервисы</string>
+    <string name="last_send">Последнее отправленное</string>
+    <string name="last_receive">Последнее полученное</string>
+    <string name="bytes_sent">Байты отправленные</string>
+    <string name="bytes_receive">Байты полученные</string>
+    <string name="time_offset">Время оффсета</string>
+    <string name="version">Версия</string>
+    <string name="sub_version">Подверсия</string>
+    <string name="inbound">Входящие</string>
+    <string name="syncode">Syncode</string>
+    <string name="peer_details">Подробнее об узле</string>
+    <string name="peers">Узлы</string>
+    <string name="no_peer_found">Не найден ни один узел</string>
+    <string name="get_peer_info">Получить информацию об узлах</string>
+    <string name="summary_peers">Просмотреть доступные узлы и информацию о них</string>
+    <string name="connecting_to_peers">Подключение к узлам</string>
+    <string name="invalid_peer_address">Адрес узла недействителен</string>
+    <string name="subscribe_to_block_notification">Подписаться на уведомления по блокам</string>
+    <string name="discover_addresses">Обнаружить адреса</string>
+    <string name="connected_to_remote_node">Подключено к удалённым нодам</string>
+    <string name="connecting_to_rpc_server">Подключается к серверу RPC</string>
+    <string name="connecting_attempt">Попытка подключения</string>
+    <string name="rpc_connection_failed">Сбой подключения к RPC:</string>
+    <string name="peer">узел</string>
+    <string name="fetched">получено</string>
+    <string name="fetching_missing_cfilters">получение недостающих CFilters</string>
+    <string name="cfilters">CFilters</string>
+    <!--/Peers-->
+
+    <!--Account-->
+    <string name="account">Аккаунт</string>
+    <string name="accounts">Аккаунты</string>
+    <string name="account_number">Номер аккаунта</string>
+    <string name="edit_account_name">Редактировать имя аккаунта</string>
+    <string name="spinner_account">Аккаунт:</string>
+    <string name="input_your_new_account_name_and_enter_your_private_passphrase">Введите имя вашего аккаунта и вашу личную парольную фразу</string>
+    <string name="account_name">Имя аккаунта</string>
+    <string name="create_account">Создать аккаунт</string>
+    <string name="add_account">Добавить аккаунт</string>
+    <string name="creating_account">Создаётся аккаунт…</string>
+    <string name="input_account_name">Введите имя для вашего эккаунта</string>
```

Comment:
the same as above: "имя" for "name" is using in case if we talk about "user name", but for "account name" is better to translate as: **Введите название для вашего аккаунта**